src/lib/config.ts

```diff
@@ -95,7 +95,6 @@ class default_config {
      * They consist of key sequences mapped to ex commands.
      */
     ignoremaps = {
-        "<S-Insert>": "mode normal",
```

Comment:
Is this supposed to be removed?

Reply:
Ah, probably not, I didn't look at the changes in config.ts (I thought they had been reverted).